pycbc/psd/read.py

```diff
@@ -187,3 +192,90 @@ def from_xml(filename, length, delta_f, low_freq_cutoff, ifo_string=None,
 
     return from_numpy_arrays(freq_data, noise_data, length, delta_f,
                              low_freq_cutoff)
+
+
+class PrecomputedTimeVaryingPSD(object):
+    def __init__(self, opt, length, delta_f, sample_rate):
```

Comment:
@bhooshan-gadre I think this is a nice way to do this, but I think it might also be useful to be able to invoke this class by hand. Would it not make sense to have a from_cli class method that uses a option/arg  input and then a regular init method that could be invoked by hand ?